Source/items.cpp

```diff
@@ -4336,11 +4309,11 @@ void SpawnHealer(int lvl)
 	SortVendor(healitem + PinnedItemCount);
 }
 
-void SpawnStoreGold()
+void InitGoldTemplate()
 {
-	golditem = {};
-	GetItemAttrs(golditem, IDI_GOLD, 1);
-	golditem._iStatFlag = true;
+	GoldTemplate = {};
+	GetItemAttrs(GoldTemplate, IDI_GOLD, 1);

```

Comment:
I think this should be `InitializeItem`,`GetItemAttrs` does an RNG call to set the gold value which never ends up being relevant for this template (I'd argue its more predictable having a value of 0).